sklearn/ensemble/tests/test_stacking.py

```diff
@@ -430,7 +429,10 @@ def test_stacking_classifier_stratify_default():
                 final_estimator=LogisticRegression(),
                 cv=KFold(shuffle=True, random_state=42),
             ),
-            *load_breast_cancer(return_X_y=True),
+            *[
+                scale(data) if idx == 0 else data
+                for idx, data in list(enumerate(load_breast_cancer(return_X_y=True)))
```

Comment:
It is too complex code here.

I think this is better to load the breast cancer dataset on the top of the file and named it `X_breast_cancer` and `y_breast_cancer` and to scale `X` there.

Reply:
Yes, definitely. Per your suggestion I have created ```X_breast_cancer``` and ```y_breast_cancer``` and used those now in the commit below

https://github.com/akaashp2000/scikit-learn/commit/3b8c7e7ac3c2b4fc324609984668bc70eeca1784